packages/web-components/fast-components/src/tabs/tabs.styles.ts

```diff
@@ -20,6 +20,8 @@ export const TabsStyles = css`
         position: relative;
         width: max-content;
         align-self: end;
+        padding: 16px 16px 0 16px;
```

Comment:
The final value is redundant if it's the same as the second (unless that's something enforced by prettier?)
```suggestion
        padding: 16px 16px 0;
```

Reply:
Nope :)